packages/blockchain-utils-linking/src/__tests__/ethereum.test.ts

```diff
@@ -165,6 +167,29 @@ describe('EthereumAuthProvider', () => {
     const auth = new ethereum.EthereumAuthProvider(provider, address)
     await expect(auth.withAddress(address).createLink(testDid)).resolves.toMatchSnapshot()
   })
+  test('accountId', async () => {
+    const address = addresses[0]
+    const auth = new ethereum.EthereumAuthProvider(provider, address)
+    const providerSpy = jest.spyOn(provider, 'sendAsync')
+    const results = []
+    // No matter how many times `::accountId` is called
+    for (let m = 0; m <= Math.floor(Math.random() * 10) + 1; m++) {
```

Comment:
I'm always very suspicious of randomness in tests unless there is a very strong rationale for doing so.  Will this test be meaningfully less valuable if we just hard coded this at, say, 10 requests?

Non-determinism in tests makes them very hard to reliably debug and reproduce failures.

Reply:
Changed it to be deterministic.